tree/dataframe/CMakeLists.txt

```diff
@@ -36,8 +27,20 @@ ROOT_STANDARD_LIBRARY_PACKAGE(ROOTDataFrame
                               HEADERS ${dictHeaders}
                               SOURCES ${sources}
                               DICTIONARY_OPTIONS "-writeEmptyRootPCM"
-                              LIBRARIES ${TBB_LIBRARIES} ${ARROW_SHARED_LIB} ${DATAFRAME_SQLITE_LIBRARIES}
+                              LIBRARIES ${TBB_LIBRARIES}
                               DEPENDENCIES Tree TreePlayer Hist RIO ROOTVecOps Imt
                               ${TREEPLAYER_DEPENDENCIES})
+if (ARROW_FOUND)
+  target_include_directories(ROOTDataFrame PRIVATE ${ARROW_INCLUDE_DIR})
+  target_link_libraries(ROOTDataFrame PRIVATE ${ARROW_SHARED_LIB})
+endif()
+
+if (sqlite)
+  target_include_directories(ROOTDataFrame PRIVATE ${SQLITE_INCLUDE_DIR})
+  target_link_libraries(ROOTDataFrame PRIVATE ${SQLITE_LIBRARIES})
```

Comment:
Shouldn't these be `PUBLIC` since `sqlite3.h` is included in `ROOT/RSqliteDS.hxx`, which is included by users? Same is probably true for the arrow equivalents above and maybe for `Davix::Davix` below if `davix.hpp` is included in a header users need.

Reply:
The header of the sqlite data source is now changed such that it doesn't include `sqlite3.h` anymore. The arrow data source header didn't include anything special in the first place.